src/Adapter/Driver/IbmDb2/Connection.php

```diff
@@ -114,7 +114,7 @@ public function connect()
                 }
             }
 
-            return;
+            return; // phpcs:ignore
```

Comment:
Again. Do not ignore coding standard issues. Fix them.

Reply:
Since return value is expected
 
```suggestion
            return null;
```

Reply:
Are you want to remove empty retrun from here? error showing `Empty return statement not required here`

Reply:
done

Reply:
There is a distinction between `return;` and `return null` or omitting return entirely.

When return types are declared both of those will result in a fatal error:

```
function (): void {
    return null;
};
```

```
function (): ?string {
    return;
};
```

In this case return type is not explicitly defined so php does not know which way is correct so there is no error.

We could technically omit return entirely for implicit return at the end.
When return value is not expected omitting `return;` is correct.
However, when return is expected but no return statement present we get a code quality issue since it is not entirely clear whether `return null` was implied or return statement was simply forgotten. Psalm normally would complain about this case.